README.md

```diff
@@ -1197,6 +1201,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Mitesh Chaudhari](https://github.com/meet931)
   - [Mithin Dev](https://github.com/mithindev)
   - [Mithun K](https://github.com/Mithun-windows-linux)
+  - [mkbhru](https://github.com/mkbhru)
```

Comment:
Hey, over here it should be `[your name](https://github.com/yourusername)` in your case: `[Manish Kumar](https://github.com/mkbhru)`, but however it is not a requirement, we could merge it if you don't want to change it 👍 Just a suggestion